src/utils/utils.ts

```diff
@@ -47,12 +88,30 @@ export const getNumItems = async (uuid: string) => {
 
     if (response.status === 200) {
       const data = await response.json();
-      const numItems = data.nyplAPI.response.numItems;
-      return numItems;
+      const responseData = data.nyplAPI.response;
+      // if (apiUrl.includes('api/v2/items/total')) {
+      //   console.log("responseData for api/v2/items/total", responseData)
+      // }
+      return responseData;
     } else {
       return 0;
     }
   } catch (error) {
     return 0;
   }
 };
+
+function addCommas(number) {
+  // Convert the number to a string
+  let numberString = number.toString();
+
+  // Use a regular expression to add commas to the string
+  numberString = numberString.replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```

Comment:
There's a native way to do this with `.toLocalString("en-US")` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString

This should be called on a number though, so you don't need to convert it to a string to call the function on the number, and it'll also return a string. Will the API always return a number for `res.count.$`? If so, then this is safe to use, but if it _might_ return a string then you'll have to add some safety checks to convert the returned value before using `number.toLocalString("en-US")`.

Reply:
The link shows an example usage in Arabic. When/if this app gets localization, maybe we can use the language code for every translation, e.g. `x.toLocaleString('ar-EG');`. Trying to forward think for an accessibility feature.